Translations/zh-Hant.lproj/advanced.strings

```diff
@@ -3,22 +3,22 @@
 "2221.title" = "壓縮單一檔案";
 
 /* Class = "NSButtonCell"; title = "Delete file(s) after compression"; ObjectID = "2222"; */
-"2222.title" = "壓縮完成後刪除原始檔案";
+"2222.title" = "壓縮完成後刪除檔案";
 
 /* Class = "NSButtonCell"; title = "Exclude Mac resource forks"; ObjectID = "2244"; */
-"2244.title" = "排除 Mac 資源檔案";
+"2244.title" = "排除 Mac 資源 fork";
 
 /* Class = "NSTextFieldCell"; title = "Password:"; ObjectID = "2262"; */
 "2262.title" = "密碼：";
 
 /* Class = "NSTextFieldCell"; title = "Split:"; ObjectID = "2270"; */
-"2270.title" = "切割：";
+"2270.title" = "分裝大小：";
```

Comment:
Not sure about this one. "Split" means cutting into pieces. This is for example the text shown in 7-Zip:

![Screenshot 2021-01-25 at 10 49 48](https://user-images.githubusercontent.com/480163/105689507-21b14500-5efb-11eb-96ef-35469cd4f213.png)

I'm using simpler and shorter texts, so "Split" will fo for me.